cyclonedx_py/parser/pipenv.py

```diff
@@ -34,6 +34,7 @@ class PipEnvParser(BaseParser):
 
     def __init__(
             self, pipenv_contents: str, use_purl_bom_ref: bool = False,
+            omit_category: Set[str] = set(),  # noqa: B006
```

Comment:
:x: do NOT ignore/suppress the bug.
FIX IT! 

this is a serious issue. Please learn how to avoid the "bug" 
and fix the implementation.

Reply:
I understand what you want. Here is the situation:
* You wanted it to be a `Set[str]`
* omit_category: Set[str] must have a default value when it appears behind use_purl_bom_ref: bool = False
* omit_category: Set[str] = set() => B006
* omit_category: Set[str] = None => error: Incompatible default for argument "omit_category"
* omit_category: Set[str] = frozenset() => incompatible to Set[str]
* omit_category: FrozenSet[str] = frozenset(str) => error: No overload variant of "frozenset" matches argument type "Type[str]"

=> no idea how to fix this correctly
=> move omit_category: Set[str] **before** use_purl_bom_ref: bool = False

Reply:
fixed